jobs/taar_lite_guidguid.py

```diff
@@ -94,18 +77,47 @@ def get_addons_per_client(broadcast_amo_whitelist, users_df):
     return df
 
 
+def get_table(view):
+    """Helper for determining what table underlies a user-facing view, since the Storage API can't read views."""
+    bq = bigquery.Client()
+    view = view.replace(":", ".")
+    # partition filter is required, so try a couple options
+    for partition_column in ["DATE(submission_timestamp)", "submission_date"]:
+        try:
+            job = bq.query(
+                f"SELECT * FROM `{view}` WHERE {partition_column} = CURRENT_DATE",
+                bigquery.QueryJobConfig(dry_run=True),
+            )
+            break
+        except Exception:
+            continue
+    else:
+        raise ValueError("could not determine partition column")
+    assert len(job.referenced_tables) == 1, "View combines multiple tables"
+    table = job.referenced_tables[0]
+    return f"{table.project}:{table.dataset_id}.{table.table_id}"
+
+
 def get_initial_sample(spark, thedate):
-    """ Takes an initial sample from the longitudinal dataset
-    (randomly sampled from main summary). Coarse filtering on:
-    - number of installed addons (greater than 1)
-    - corrupt and generally wierd telemetry entries
-    - isolating release channel
-    - column selection
     """
+    Get a DataFrame with a valid set of sample to base the next
+    processing on.
+
+    Sample is limited to submissions received since `date_from` and latest row per each client.
+
+    Reference documentation is found here:
+
+    Firefox Clients Daily telemetry table
+    https://docs.telemetry.mozilla.org/datasets/batch_view/clients_daily/reference.html
+
+    BUG 1485152: PR include active_addons to clients_daily table:
+    https://github.com/mozilla/telemetry-batch-view/pull/490
+    """
+
     # Could scale this up to grab more than what is in
     # longitudinal and see how long it takes to run.
     s3_url = "gs://moz-fx-data-derived-datasets-parquet/clients_daily/v6/submission_date_s3={}".format(
```

Comment:
nit: change var name from s3_url to gcs_url

Reply:
renamed